apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -0,0 +1,314 @@
+  Given the behavior with no automatic linking, email addresses are no longer a unique identifier for a user account. Please always use the user's UUID to correctly reference user accounts.
+- **Sessions may have a maximum duration.**  
+  Depending on the configuration of the identity provider, a login session established with SSO may forcibly log out a user after a certain period of time.
+
+### Row Level Security
+
+You can use information about the SSO identity provider in Row Level Security policies.
+
+Here are some commonly used statements to extract SSO related information from the user's JWT:
+
+- `auth.jwt()#>>'{amr,0,method}'`
+  Returns the name of the last method used to verify the identity of this user. With SAML SSO this is `sso/saml`.
+- `auth.jwt()#>>'{amr,0,provider}'`
+  Returns the UUID of the SSO identity provider used by the user to sign-in.
+- `auth.jwt()#>>'{user_metadata,iss}'`
+  Returns the identity provider's SAML 2.0 `EntityID`
+
+<Admonition type="caution">
+If you use [Multi-Factor Authentication](/guides/auth/auth-mfa) with SSO, the `amr` array may have a different method at index `0`!
+</Admonition>
+
+A common use case with SSO is to use the UUID of the identity provider as the identifier for the organization the user belongs to. This way you can use restrictive RLS policies to scope down the SQL actions and data that a user is able to access based on what SSO provider was used to sign the user in.
+
+For example, let's say you have a table like:
+
+```sql
+create table organization_settings (
+  -- the organization's unique ID
+  id uuid not null primary key,
+  -- the organization's SSO identity provider
+  sso_provider_id uuid unique,
+  -- name of the organization
+  name text,
+  -- billing plan (paid, free, enterprise)
+  billing_plan text
+);
+```
+
+You can use the information present in the user's JWT to scope down which rows from this table the user can see, without doing any additional user management:
+
+```sql
+create policy "View organization settings."
+  on organization_settings
+  as restrictive
+  using (
+    sso_provider_id = auth.jwt()#>>'{amr,0,provider}'
+  );
+```
+
+## Managing SAML 2.0 connections
+
+### Prerequisites
+
+SSO support with SAML 2.0 is in an early beta release. This guide uses the following software which you need to install on your machine to configure your project:
+
+- [**cURL**](https://curl.se)  
+  It is typically pre-installed in macOS and GNU/Linux distributions.
+- [**jq**](https://stedolan.github.io/jq/)  
+  You can install it with `brew install jq` on macOS or using your distribution's package manager.
+
+You would need access to two keys -- the `anon` and `service_role` key. You can obtain these on the [Project API Keys](https://app.supabase.com/project/_/settings/api) page in the dashboard.
+
+We publish an [OpenAPI specification](https://github.com/supabase/gotrue/blob/master/openapi.yaml) which you can refer to at any time.
```

Comment:
Obviously all of this will be replaced with well designed `supabase` CLI commands.